src/Mailgun/Resource/Api/Suppressions/Complaint/ShowResponse.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions\Complaint;
+
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class ShowResponse implements ApiResponse
+{
+    /**
+     * @var string
+     */
+    private $address;
+
+    /**
+     * @var string
+     */
+    private $created_at;
+
+    /**
+     * @param string $address
+     * @param string $code
+     * @param string $error
+     */
+    public function __construct($address)
+    {
+        $this->address = $address;
+        $this->created_at = new \DateTime().format(\DateTime::COOKIE);
```

Comment:
This is a PHP SDK, ideally we should manage a `DateTime` object so the developer can manage it as required based on his/her code